ccds/__main__.py

```diff
@@ -31,6 +33,11 @@ def _main(*args, **kwargs):
         f.params[1].default = (
             "https://github.com/drivendataorg/cookiecutter-data-science"
         )
+        # Find the "checkout" option in the cookiecutter cli (currently the fifth)
+        # Per #389, set this to the currently released version by default
+        param_names = [p.name for p in f.params]
+        checkout_index = param_names.index("checkout")
+        f.params[checkout_index].default = __version__
```

Comment:
It looks like some places (like the release tags), we use `vX.Y.Z` and others we use `X.Y.Z`. We should check if we need the `v` prefix here or not.

Reply:
@pjbull great catch, would have failed as-is without the "v" prefix. I added a `RELEASING.md` with some instructions, but feel free to tweak or make suggestions.